code/drasil-lang/Language/Drasil/DisplayExpr.hs

```diff
@@ -21,3 +22,4 @@ data DisplayExpr where
     SpaceExpr     :: Space -> DisplayExpr
     BinOp         :: DisplayBinOp -> DisplayExpr -> DisplayExpr -> DisplayExpr
     AssocBinOp    :: DisplayAssocBinOp -> [DisplayExpr] -> DisplayExpr
+    ForAll        :: UID -> Space -> DisplayExpr
```

Comment:
With "ForAll"/universal quantification, we're saying that "given any variable that belongs to a space, this predicate function/relation will hold". Using [GADT syntax](https://en.wikibooks.org/wiki/Haskell/GADT#GADTs), the last component of the signature is the resulting type of the constructor, not a parameter of the constructor. So, it ends up that we're missing an extra DisplayExpr (for the predicate function) after the Space argument.